src/i18n/lang/fr.json

```
@@ -732,6 +732,7 @@
       "validator": "Validant",
       "toValidate": "A valider avant",
       "errorDate": "Date invalide",
+      "exempleDate": "(Ex: 31/05/1970)",
```

```suggestion
      "exempleDate": "(Ex: 31/05/2022)",
```
Histoire de pas faire trop vieux x)